omas/tests/machine_mappings/test_data/d3d_ECRH_machine_mapping.json

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
@AreWeDreaming we have been placing these files under `omas/samples`
Also, you could update `omas/omas_sample.py` to support `ods.sample("ec_launchers")`

Reply:
Do you mean edit the `omas_core.py` function sample?
```
sample(self, ntimes=1, homogeneous_time=None)
```
To something like:
```
sample(self, pr="All", ntimes=1, homogeneous_time=None)
```
I have added the function `ec_launchers` to `omas_sample.py` and then run `omas.sample(1)` to fetch all sample data including the `ec_launchers`.